dashboard/db/migrate/20160213001925_add_submitted_to_user_levels.rb

```diff
@@ -0,0 +1,5 @@
+class AddSubmittedToUserLevels < ActiveRecord::Migration
+  def change
+    add_column :user_levels, :submitted, :boolean
+  end
```

Comment:
I think if it takes more than an hour we can't do it or we need to think of an another approach. We can avoid impacting the reporting read replica by creating a new read replica (this is easy, it's literally pushing a button in AWS).